dotcms-postman/src/main/resources/postman/Workflow_Resource_Tests.json

```diff
@@ -16965,6 +16965,194 @@
 					"response": []
 				}
 			]
+		},
+		{
```

Comment:
I think a test with something like:


```
curl --location --request PUT 'http://localhost:8080/api/v1/workflow/actions/default/fire/NEW' \
--header 'Content-Type: application/json' \
--data '{
    "contentlet": {
        "contentType": "Product",
        "title": "supporttest",
        "titleFeature": "{\"12345":"d=2024-06-20&d2=2024-06-21&c=prc&v=56\"},
        "titleEurope": "{\"12345":"d=2024-06-20&d2=2024-06-21&c=prc&v=56\"}
    }
}'
```

is missing, and I think it is supported by the changes here:

https://github.com/dotCMS/core/pull/28936/files#diff-fa1ceaa19618a6b2bbc30e24c6f930b4971f417db50babb748c2e2837ba9eb82R7239